plugins/richcombo/plugin.js

```diff
@@ -381,6 +383,19 @@ CKEDITOR.plugins.add( 'richcombo', {
 					this._.lastState = this._.state;
 					this.setState( CKEDITOR.TRISTATE_DISABLED );
 				}
+			},
+			/**
+			 * Removes all listeners from richCombo element.
+			 *
+			 * @since 4.9.0
+			 */
+			destroy: function() {
+				if ( this._listeners.length ) {
```

Comment:
IMO this check is totally unnecessary. `forEach` loop would work totally fine with empty array.